doc/docs/Documentation/User/index.md

```diff
@@ -2,4 +2,33 @@
 icon: material/book-open-variant 
 ---
 
-# **Coming soon**
\ No newline at end of file
+# User manual
+
+Figure 1 show a schematic overview of the `SLOTH` features.
+
+<figure markdown="span">
+  ![sloth-features](../../img/sloth-diagram.png){  width=500px}
+  <figcaption>Figure 1 : schematic overview of the `SLOTH` features
+</figcaption>
+</figure>
+
+`SLOTH` is developed on top of `MFEM` [@mfem].
+It is implemented in C++ and the source code is managed with CMake. 
```

Comment:
source code is managed with CMake. 
-> software build process is managed with CMake